custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -49,14 +47,18 @@
         <script src="{% static 'angular-leaflet-directive/dist/angular-leaflet-directive.js' %}"></script>
         <script src="{% static 'perfect-scrollbar/js/perfect-scrollbar.jquery.min.js' %}"></script>
         <script src="{% static 'angular-perfect-scrollbar/src/angular-perfect-scrollbar.js' %}"></script>
-
         <script src="{% static 'topojson/topojson.js' %}"></script>
         <script src="{% static 'datamaps/dist/datamaps.ind.js' %}"></script>
-
+{% endcompress %}
```

Comment:
Not blocking, but a comment here noting why these **aren't** compressed might be helpful.

Reply:
Good idea: https://github.com/dimagi/commcare-hq/pull/25279/commits/57843474ed259d703424a3f72b8ec5942efef221